StudyBuddy/Controllers/ProfileController.cs

```diff
@@ -31,13 +35,13 @@ public IActionResult DisplayProfiles()
 
     public IActionResult UserId(string id)
     {
-        UserId parseUserId= (UserId)Guid.Parse(id);
+        UserId parseUserId = (UserId)Guid.Parse(id);
```

Comment:
Please use 
```cs
UserId parseUserId = UserId.From(Guid.TryParse(id));
```

Casting not very safe, also Parse might fail so using TryParse is a good ease of mind fix.

Reply:
👉👈👍